bagua/bagua_define.py

```diff
@@ -37,6 +37,7 @@ class BaguaHyperparameter(BaseModel):
     """
 
     buckets: List[List[TensorDeclaration]] = []
+    bucket_size: int = 0
```

Comment:
Is this hyperparameter class only used in service?

Reply:
Yes, I put it here because this is the simplest and most intuitive way to achieve

Reply:
I mean the `BaguaHyperparameter` class. If it is only used in service, we can move the whole class to service module

Reply:
Not exactly, new distributed hyperparameters should also be configured here, such as `is_hierarchical_reduce`